tests/TestOne/buildtool/bin/buildtool

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+#
+# This is a very simple pjc-aware ocamlfind-based build tool.
+
+# Build
+
+DEPENDS=""
+for p in $cur_depends; do
+  if [ "$p" != "buildtool" ]; then
+    DEPENDS="$DEPENDS $p"
+  fi
+done
+
+echo "Build location: $cur_target_dir"
+
+mkdir -p $cur_target_dir/lib
+mkdir -p $cur_target_dir/bin
+
+## Prepare ocamlfind options
+
+OCAMLFIND_ARGS=""
+
+for p in $DEPENDS; do
+  OCAMLFIND_ARGS="$OCAMLFIND_ARGS -package $p"
+done
+
+## Build modules
+
+for src in `find $cur_root/lib -name "*.ml"`; do
+  ocamlfind ocamlc -c $OCAMLFIND_ARGS $src -o $cur_target_dir/lib/`basename $src`
```

Comment:
This would need to use `ocamldep` to make sure they're compiled in the right order, but this is just meant to be a dirt simple example so I wouldn't worry about it.